docs/js-core-1/week-3/homework.md

```diff
@@ -4,92 +4,60 @@ title: Coursework
-- Add Items to an Array with push() and unshift()
-- Remove Items from an Array with pop() and shift()
-- Remove Items Using splice()
-- Add Items Using splice()
-- Copy Array Items Using slice()
-- Copy an Array with the Spread Operator
-- Combine Arrays with the Spread Operator
-- Check For The Presence of an Element With indexOf()
-- Iterate Through All an Array's Items Using For Loops
-- Create complex multi-dimensional arrays
-- Add Key-Value Pairs to JavaScript Objects
-- Modify an Object Nested Within an Object
-- Access Property Names with Bracket Notation
-- Use the delete Keyword to Remove Object Properties
-- Check if an Object has a Property
-- Iterate Through the Keys of an Object with a for...in Statement
-- Generate an Array of All Object Keys with Object.keys()
-- Modify an Array Stored in an Object
+Write down your target areas and take them to your buddy group. What does your mentor think? Do they agree with your assessment?
 
 ## 3) JavaScript Exercises (3 hours)
 
-Here are a set of exercise that will help you to solidify your knowledge of JavaScript.
+These exercises will help solidify your knowledge of JavaScript. Open the "Exercises" folder and complete all of the challenges.
 
-For this step, open the "Exercises" folder and complete all of the challenges.
+Before you start, **make sure you fork the repository** to your Github account.
 
 https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week3
 
-Before you start, **make sure you fork the repository** to your Github account.
-
 ## 4) JavaScript Challenges (10 hours)
 
 Next, in the same repository complete all of the exercises in the "Mandatory" folder.
 
 https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week3
 
-## 5) Give feedback on your colleague's coursework (1 hour)
-
-Just like last week, we want you to continue to give good, helpful feedback to the other trainees in your class.
-
-You should
+## 5) Code review in your Buddy Group
 
-1. View the open Pull Requests here
+Code review is a valuable skill for a developer, and is a good way to practice talking about code with confidence. Let's level up our code review now.
 
-https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week3/pulls
-
-2. Find **two** pull requests from people in your class that have no, or few, comments on them
-3. Leave comments on the **two** pull requests with improvements or suggestions
-4. Leave a comment saying that you have reviewed it and a nice comment about something you liked (always remember to be nice!)
-
-Make sure you mark at least two other pull requests. It's important that you see a variety of different answers!
-
-Remember, you should use the Code Style Guide to help you
-
-https://syllabus.codeyourfuture.io/guides/code-style-guide
+1. Read Google's [engineering guide on code review](https://google.github.io/eng-practices/review/reviewer/comments.html#:~:text=Be%20kind.,explaining%20the%20complexity%20to%20you.).
```

Comment:
This was also linked in week 1 - maybe this should be "refresh yourself on" rather than "read"?

Reply:
fixed in #ba68b1d